tests_integration/__tests__/patterns-dirs.js

```diff
@@ -120,12 +120,11 @@ if (path.sep === "/") {
 }
 
 function testPatterns(namePrefix, cliArgs, expected = {}) {
-  const testName =
-    (namePrefix ? namePrefix + ": " : "") +
-    "prettier " +
-    cliArgs
-      .map((arg) => (/^[\w./=-]+$/.test(arg) ? arg : `'${arg}'`))
-      .join(" ");
+  const testName = `${
+    namePrefix ? `${namePrefix}: ` : ""
+  }prettier ${cliArgs
+    .map((arg) => (/^[\w./=-]+$/.test(arg) ? arg : `'${arg}'`))
+    .join(" ")}`;
```

Comment:
This is so ugly.

Reply:
Make it _Prettier_ in another PR :)

Reply:
That's why I'm against enforcing lint rules like this. Often enough concatenation is more readable than a template literal, and that can't be fixed by changing the formatting of the template literal.

"Linters are here to help people find bugs." or "People are here to make linters happy."? What's your camp? ;)

Reply:
Of course can't win them all.. but string literals are generally better. I'm sure in this case could make it better if using some variables

Reply:
Often better but now always. E.g. for simple things like `someLongExpression(withArg) + "\n"` concatenation is more clear too because `+` expresses the same idea of addition as in `expr + 1`: "add one to a number", "add `\n` to a string". Template literals are clear winners only if the static part of the template is bigger than the expressions.

Reply:
Not always, for example:
```js
const links = `Docs: ${links.map((link) => colors.bold(link)).join('\n')};
```

Expression is more, yep it can be refactored on separate variables, but why?

Reply:
I mean in those situations they're **clear** winners. In other situations things aren't as clear-cut.

Reply:
Overall, for strings with leading/trailing spaces, templates are usually better for this reason:
```js
const links = "Docs: " + links.map((link) => colors.bold(link)).join('\n');
//                  ^^^ 👎
const links = `Docs: ${links.map((link) => colors.bold(link)).join('\n')}`;
//                  ^ 👍
```